src/screens/CreatePost/index.js

```diff
@@ -778,6 +756,8 @@ const CreatePost = () => {
                 setIsPollMultipleChoice(ismultiplechoice)
               }
               expiredobject={postExpired[expiredSelect].expiredobject}
+              isAnonym={typeUser}
+              expiration={postExpired[expiredSelect].label}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isAnonym` and `expiration` props could potentially improve the functionality of the `PollItem` component by providing more information. However, ensure that these props are correctly used inside the `PollItem` component.
```diff
+ isAnonym={typeUser}
+ expiration={postExpired[expiredSelect].label}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->